ArduCopter/version.cpp

```diff
@@ -0,0 +1,16 @@
+#include "version.h"
+
+#include "GCS_MAVLink/GCS.h" // for FIRMWARE_VERSION_TYPE_DEV and friends
+
+const AP_FWVersion fwver
```

Comment:
This is more a matter of trying to make things better, not perfect.

So I also resisted the temptation (to date) of making the thing a full-blown class... to make it a member of Copter I think you might want to do that way and have a constructor taking the components and what-not.  That same class could allow passing in a buffer to retrieve the FIRMWARE_STRING and that sort of gumpf.